pom.xml

```diff
@@ -18,10 +18,56 @@
         <maven>3.3.0</maven>
     </prerequisites>
 
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>1.8</java.version>
```

Comment:
The ANTLR module uses Java version 1.7. Is this intentional ?

Reply:
The ANTLR module is no longer under active development but the ANTLR-based Querqy query parser is used in a number of tests. Thank you for spotting this and updating the Java version.